client/state/plugins/installed/selectors-ts.ts

```diff
@@ -446,3 +446,25 @@ export const getPluginStatusesByType = createSelector(
 	},
 	( state: AppState ) => state.plugins.installed.status
 );
+
+/**
+ * Returns a Boolean indicating if a site has a particular plugin that
+ * is active. This is useful for jetpack connected sites.
```

Comment:
Minor suggestion for improving clarity:

```suggestion
 * Returns true if a particular plugin is installed and active for a specified site.
 * This is useful for Jetpack connected sites.
```